src/test/java/org/jboss/tools/intellij/openshift/utils/odo/JsonParserTest.java

```diff
@@ -87,26 +87,26 @@ public void verifyThatURLSCanBeLoadedFromDeployModeOnOpenShift() throws IOExcept
     assertEquals("/", urls.get(0).getPath());
   }
 
-    @Test
-    public void verifyThatStarterCanBeLoadedFromRegistry() throws IOException {
-        URL url = JsonParserTest.class.getResource("/devfile-registry.json");
-        JSonParser parser = new JSonParser(MAPPER.readTree(url));
-        ComponentTypeInfo componentTypeInfo = parser.parseComponentTypeInfo();
-        assertNotNull(componentTypeInfo);
-        assertEquals(1, componentTypeInfo.getStarters().size());
-        assertEquals("dotnet50-example", componentTypeInfo.getStarters().get(0).getName());
-        assertNull(componentTypeInfo.getStarters().get(0).getDescription());
-    }
+  @Test
+  public void verifyThatStarterCanBeLoadedFromRegistry() throws IOException {
+    URL url = JsonParserTest.class.getResource("/devfile-registry.json");
+    JSonParser parser = new JSonParser(MAPPER.readTree(url));
+    ComponentTypeInfo componentTypeInfo = parser.parseComponentTypeInfo();
+    assertNotNull(componentTypeInfo);
+    assertEquals(1, componentTypeInfo.getStarters().size());
+    assertEquals("dotnet50-example", componentTypeInfo.getStarters().get(0).getName());
+    assertNull(componentTypeInfo.getStarters().get(0).getDescription());
+  }
 
-    @Test
-    public void verifyThatComponentInfoCanBeLoadedFromDescribe() throws IOException {
-        URL url = JsonParserTest.class.getResource("/describe-component-local.json");
-        JSonParser parser = new JSonParser(MAPPER.readTree(url));
-        ComponentInfo componentInfo = parser.parseDescribeComponentInfo(ComponentKind.DEVFILE);
-        assertNotNull(componentInfo);
-      assertTrue(componentInfo.getSupportedFeatures().contains(ComponentFeature.Mode.DEV_MODE));
-        assertEquals("Go", componentInfo.getComponentTypeName());
-        assertEquals("Go", componentInfo.getLanguage());
-    }
+  @Test
+  public void verifyThatComponentInfoCanBeLoadedFromDescribe() throws IOException {
+    URL url = JsonParserTest.class.getResource("/describe-component-local.json");
+    JSonParser parser = new JSonParser(MAPPER.readTree(url));
+    ComponentInfo componentInfo = parser.parseDescribeComponentInfo(ComponentKind.DEVFILE, false);
+    assertNotNull(componentInfo);
+    assertTrue(componentInfo.getSupportedFeatures().contains(ComponentFeature.Mode.DEV_MODE));
+    assertEquals("Go", componentInfo.getComponentTypeName());
+    assertEquals("Go", componentInfo.getLanguage());
+  }
 
```

Comment:
Maybe you could add a unit test in `DevOnPodmanComponentActionTest` testing enablement when podman is present/not present?